### DISCUSSION TITLE
Themes and styling

### DISCUSSION THREAD

**Discussion by Netzeband** [thaw-ui/thaw#288] [Q&A]
Hello,

I'm new to leptos and I'm absolutely not a web designer. I noticed, that the components of thaw ui are a little bit "colorless" and there is no documentation about styling or themes at all. So how to do this?

Actually I don't care so much about the specific style, but just having black/white components is a little bit pale. My expectation would be to have somewhere a theme css file, where I define a handful of colors (like primary, secondary, background and so on), which are then used consistently across all components. 

Did I miss something here?

**Comment by lizidev** [comment]
Currently you can customize the theme by modifying the theme property of the ConfigProvider component. https://thawui.vercel.app/components/config-provider#customize-theme

**Reply by Netzeband** [reply]
Hey, thanks for the answer. I see, I have overseen it because it is in the "components" sections of the documentation and does not have style or theme in the name. 

But is there a documentation about the different colors defined in the struct and their semantic meaning?

**Comment by lizidev** [comment]
Please refer to https://react.fluentui.dev/?path=/docs/theme-colors--docs

As for the specific meaning of the colors, there is currently no documentation.

**Reply by Netzeband** [reply]
Thanks a lot.
